Final Project Submission/Franklin_EscapeRoomGame.cpp

```diff
@@ -0,0 +1,98 @@
+
+//Escape Room Game
+//-text-based adventure
+
+//This program will be a game that continually takes user inputs as commands and translates
+//them into things that will happen in the Game
+
+#include <iostream>
+#include <fstream>
+
+using namespace std;
+
+int selectDifficulty();
```

Comment:
It would be good here to summarize what each of these functions does, as a "presentation" of the "interfaces"